fdbcli/MetaclusterCommands.actor.cpp

```diff
@@ -45,6 +45,7 @@ parseClusterConfiguration(std::vector<StringRef> const& tokens,
                           int endIndex) {
 	Optional<metacluster::DataClusterEntry> entry;
 	Optional<ClusterConnectionString> connectionString;
+	Optional<metacluster::AutoTenantAssignment> autoTenantAssignment;
```

Comment:
nit: We could assign directly to the entry and not use this intermediate

Reply:
Done